stacks-signer/src/v0/signer_state.rs

```diff
@@ -95,13 +92,24 @@ impl LocalStateMachine {
         db: &SignerDb,
         client: &StacksClient,
         proposal_config: &ProposalEvalConfig,
+        eval: &GlobalStateEvaluator,
+        local_address: &StacksAddress,
```

Comment:
Can we just use `client.stacks_address` here instead of having to provide an arg?